src/Innertube.ts

```diff
@@ -452,6 +455,82 @@ export default class Innertube {
     return response.endpoint;
   }
 
+  /**
+   * Gets a post page given a post id and the channel id
+   */
+  async GetPost(postId: string, channelId: string) : Promise<Feed<IBrowseResponse>> {
+    throwIfMissing({ postId, channelId });
+    const writer = CommunityPostParams.encode({
+      f0: 'community',
+      f1: {
+        postId: postId
+      },
+      f2: {
+        p1: 1,
+        p2: 1 
+      }
+    });
+
+    const params = encodeURIComponent(u8ToBase64(writer.finish()));
+
+    const browse_endpoint = new NavigationEndpoint({ browseEndpoint: { browseId: channelId, params: params } });
+
+    const response = await browse_endpoint.call(this.#session.actions, { parse: true });
+    return new Feed(this.actions, response);
+  }
+
+  /**
+   * Get comments for a community post.
+   */
+  async GetPostComments(postId: string, channelId: string, sort_by?: 'TOP_COMMENTS' | 'NEWEST_FIRST'): Promise<Comments> {
+    throwIfMissing({ postId, channelId });
+
+    const SORT_OPTIONS = {
+      TOP_COMMENTS: 0,
+      NEWEST_FIRST: 1
+    };
+
+    const writer1 = CommunityPostCommentsParam.encode({
+      title: 'community',
+      postContainer: {
+        postId: postId
+      },
+      f0: {
+        f0: 1,
+        f1: 1
+      },
+      commentDataContainer: {
+        title: 'comments-section',
+        commentData: {
+          sortBy: SORT_OPTIONS[sort_by || 'TOP_COMMENTS'],
+          f0: 1,
+          channelId: channelId,
+          postId: postId
+        }
+      }
+    });
+
+    const writer2 = CommunityPostCommentsParamContainer.encode({
+      f0: {
+        location: 'FEcomment_post_detail_page_web_top_level',
+        protoData: encodeURIComponent(u8ToBase64(writer1.finish()))
```

Comment:
A protobuf inside a protobuf is used here which is why there's two different writers